test/python/sdpa/frost/test_sdpa_fwd_fp8_sm100.py

```
@@ -1070,3 +1070,218 @@ def test_fp8_sm100_device_scales_execute_reads_no_device_memory():
+    _check(out, o_ref, torch.float16, "e4m3", a_o, a_o_ref)
+    diff = (lse.float() - lse_ref).abs().max().item()
+    assert diff <= 5e-2, f"max|LSE-ref| = {diff:.4f}"
+
+
+@pytest.mark.L0
+@_skip_d512_on_rubin
+@torch_fork_set_rng(seed=0)
+def test_fp8_d512_thd_zero_len_kv():
+    """THD on d512 with a zero-length KV sequence: those Q rows are dead
+    (O := 0, LSE := -inf) and must not poison amax_o."""
+    out, o_ref, a_o, a_o_ref, lse, lse_ref = _run_thd([126, 40, 60], [0, 83, 77], 8, 8, "e4m3", scale=_D512_SCALE, stats=True, d=_D512)
```

Ruff (0.16.3)

[warning] Unpacked variable `lse_ref` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)